docs/servo_power_test/src/ui.ts

```diff
@@ -78,7 +87,19 @@ export class Ui {
     this.dutConsole.textContent += s;
     this.dutConsole.scrollTo(0, this.dutConsole.scrollHeight);
   }
-  public hideToolTip() {
-    this.toolTip.classList.add('hidden');
+  public hideElement(element: HTMLElement) {
+    element.classList.add('hidden');
+  }
+  public showElement(element: HTMLElement) {
+    element.classList.remove('hidden');
+  }
+  public appendItrSelectors(iterationNumber: number, selectedIndex: number) {
+    for (let i = 0; i < iterationNumber; i++) {
+      const newOption = document.createElement('option');
+      newOption.innerText = `${i + 1}`;
+      this.iterationSelector.add(newOption);
+    }
+    this.iterationSelector.selectedIndex = selectedIndex;
+    this.showElement(this.iterationSelector);
```

Comment:
Is the iteration selector hidden by default so the showElement call is needed here? If so, adding a comment that explains so would help the readers to understand the code ;) (since show and hide is mostly paired usually, I was curious why it only calls show here...)

Reply:
you're right. Added a comment.